paper.tex

```diff
@@ -1109,84 +1109,110 @@ \section*{ARG simplification}
+of a single node that has one inbound and one outbound edge, such as
+nodes \textsf{F} and \textsf{H}.
 Fig.~\ref{fig-simplification}B shows the result of this type of
 graph topology simplification.
 
 Simplifying away diamonds will remove many unary nodes from the
-local trees, there can still be nodes that are unary in all
-of the local trees. For example, node $H$ is one of the
-parents of the recombinant node $D$ is still present in
-Fig.~\ref{fig-simplification}B after diamond removal, but
-it is unary in all of the local trees since it has an
-in-degree of $1$.
-Fig.~\ref{fig-simplification}C shows the resulting graph
-after removing these nodes.
+local trees, but there can still be nodes that are unary in all
+of the local trees. In particular, a node can represent a recombinant
+with multiple parents in the graph but only a single child (e.g. node \textsf{N}
+in Fig.~\ref{fig-simplification}B), or can represent a common ancestor with
+multiple children in the graph but in which no local coalescence takes place
+(node \textsf{R} in Fig.~\ref{fig-simplification}B).
+% point out that this is why Hudson refers to CA nodes, not coalescence nodes
+Such nodes are not singly connected in the graph, but are nevertheless unary in
+all of the local trees in Fig.~\ref{fig-simplification}B. The operation to remove them,
+the results of which are shown in Fig.~\ref{fig-simplification}C,
+therefore requires knowledge not just of the graph topology, but also of the
+edge annotations. Removal of recombinant nodes can produce graph nodes with
+more than 2 parents (e.g. node \textsf{E}); likewise, removal of
+common ancestor but non-coalescent nodes can produce graph nodes with
+more than 2 children (e.g. node \textsf{S}). These representing a ``stacking up'' of multiple
+often indistinguishable events into a single node, something which is impossible
+to encode in an eARG.
 
 The remaining nodes are MRCAs of some subset of the samples
 at \emph{some} positions along the genome. We still have
 some unary nodes in the local trees, but these nodes will
 correspond to a coalescence in at least one other
-local tree. For example, node $E$ is unary in the first tree
-of Fig.~\ref{fig-simplification}E, but is either binary
-or ternary in the other trees (recall this is a Wright-Fisher
-simulation).
-
-The final level of simplification is to remove \emph{all}
-unary nodes from the local trees (Fig.~\ref{fig-simplification}D
-and E). [And continue]
+local tree. For example, node  \textsf{K} is unary in the second tree
+of Fig.~\ref{fig-simplification}C, but is either binary
+or ternary in all subsequent trees (recall this is a Wright-Fisher
+simulation). The final level of simplification is to alter the edge annotations
+such that, although no nodes are removed from the graph, \emph{all}
+unary nodes disappear from the local trees (Fig.~\ref{fig-simplification}D).
+It is common for this operation to remove nodes above the top MRCA node
```

Comment:
I'd rephrase this a bit, but can wait for another round of revisions.

Reply:
Yeah - I put in the bit about roots with the aim of rephrasing later, but though it worth adding in case we did want to say something.